blocks/library/paragraph/index.js

```diff
@@ -117,21 +144,63 @@ class ParagraphBlock extends Component {
 			),
 			isSelected && (
 				<InspectorControls key="inspector">
-					<PanelBody title={ __( 'Text Settings' ) }>
+					<PanelBody title={ __( 'Text Settings' ) } className="blocks-font-size">
+						<div className="blocks-font-size__main">
+							<ButtonGroup aria-label={ __( 'Font Size' ) }>
```

Comment:
Won't we want `aria-label` on each of the button options instead for "Small", "Medium", etc. ?

Reply:
We discussed this and opted for not having one for now.